assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py

```diff
@@ -160,9 +160,6 @@ class ModelImportConstants:
                 "model_hf_load_kwargs": {
                     "trust_remote_code": True,
                 },
-                "tokenizer_config": {
```

Comment:
Why we removed this?

Reply:
It is not required for latest falon models with model_type=falcon, needed for older falcon models where model_type=RefinedWebModel